src/components/FeatureCard/FeatureCard.tsx

```diff
@@ -0,0 +1,114 @@
+import React from 'react';
+import { ResponsiveImage, TextGroup } from '..';
+import styles from './FeatureCard.scss';
+
+const responsiveImagePropsMapping = {
+	small: {
+		fallbackWidth: 220,
+		fallbackHeight: 122,
+		sizes: '220px',
+		widthRange: [ 220, 220 ],
+	},
+	large: {
+		fallbackWidth: 220,
+		fallbackHeight: 122,
+		sizes: `
+			(min-width: 1200px) 666px,
+			(min-width: 768px) 346px,
+			220px,
+		`,
+		widthRange: [ 220, 666 ],
+	},
+	smallPortrait: {
+		fallbackWidth: 285,
+		fallbackHeight: 400,
+		sizes: '285px',
+		widthRange: [ 285, 400 ],
+	},
+	largePortrait: {
+		fallbackWidth: 285,
+		fallbackHeight: 400,
+		sizes: `
+			(min-width: 1200px) 382px,
+			285px,
+		`,
+		widthRange: [ 285, 400 ],
```

Comment:
according to `sizes` this will never be larger than 382px wide

Reply:
Oops; brain saw "400" height and plugged that in where it shouldn't go